doc/source/commands/runprogram.rst

```diff
@@ -0,0 +1,226 @@
+.. _running:
+
+Running Programs
+================
+
+.. contents:: Contents
+    :local:
+    :depth: 2
+
+KerboScript supports running executing code saved in files on the local or
+archive disks. The terms "program" and "script" are generally used
+interchangably when describing this behavior. Programs can either be run using
+one of the :ref:`runfunctions` or the :ref:`run`.  Supported file types are:
```

Comment:
So it's not possible to link to :ref:`run` without it adding the word "keyword"?  I can think of cases where I'd want the link but not that added word.

"try typing [run] here..." -> turns into "try typing [run keyword] here." if I make [run] a link.